backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/CanonicalJson.kt

```diff
@@ -89,6 +91,10 @@ class CanonicalJson {
             }
         }
 
+        fun koblenzUserToString(koblenzUser: KoblenzUser): String {
+            return ObjectMapper().writeValueAsString(koblenzUser)
```

Comment:
We need a canonical JSON string (RFC 8785) -- I don't think Jackson guarantees this ([yet?](https://github.com/FasterXML/jackson-core/issues/1037)).
I recommend converting `koblenzUser` to a `Map<String, Any?>` manually here, and calling `serializeToString` with the map.

Reply:
Do we also have to make ensure lower or uppercase of the strings?

Reply:
I changed it to using a Map and the serializeToString function, result stays the same.